test/model/test_node.py

```diff
@@ -205,7 +217,7 @@ async def test_get_value_metadata(node, uuid4, mock_command):
         },
     )
 
-    value_id = "52-32-00-targetValue-00"
+    value_id = "52-32-0-targetValue-00-00"
```

Comment:
Do you think the endpoint fix has been a problem somewhere in production code?

Reply:
I don't believe the endpoint change has caused problems because it was consistent (values on endpoint 0 would always use `00`, and in HA, we fallback on primary value endpoint if it's not provided), but it is a breaking change that will need to be accounted for when we do a version bump

Reply:
We changed the interface of this function at least so it's good to call it out.